From 346b7b6b4931fc6bee9e820e0160dd024a86ed52 Mon Sep 17 00:00:00 2001
From: Markus Boehme <markus.boehme@mailbox.org>
Date: Wed, 27 Aug 2025 22:49:29 +0200
Subject: [PATCH] pkgconf: expose variables for system-alloc-{uid,gid}-min

Expose variables for system-alloc-uid-min and system-alloc-gid-min
similar to the UID/GID ranges already exposed for the respective
maximums, and other UID/GID ranges.
---
 src/core/systemd.pc.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/core/systemd.pc.in b/src/core/systemd.pc.in
index 58f2463104..dcf7bbc9a3 100644
--- a/src/core/systemd.pc.in
+++ b/src/core/systemd.pc.in
@@ -90,8 +90,12 @@ modulesloaddir=${modules_load_dir}
 catalog_dir=${prefix}/lib/systemd/catalog
 catalogdir=${catalog_dir}
 
+system_alloc_uid_min={{SYSTEM_ALLOC_UID_MIN}}
+systemallocuidmin=${system_alloc_uid_min}
 system_uid_max={{SYSTEM_UID_MAX}}
 systemuidmax=${system_uid_max}
+system_alloc_gid_min={{SYSTEM_ALLOC_GID_MIN}}
+systemallocgidmin=${system_alloc_gid_min}
 system_gid_max={{SYSTEM_GID_MAX}}
 systemgidmax=${system_gid_max}
 
-- 
2.51.0

